src/app/legacy/containers/PodcastPromo/index.stories.jsx

```diff
@@ -31,15 +32,17 @@ const serviceContextMock = {
 
 // eslint-disable-next-line react/prop-types
 const Component = ({ inline = false }) => (
-  <ToggleContextProvider
-    toggles={{
-      eventTracking: { enabled: true },
-    }}
-  >
-    <ServiceContext.Provider value={serviceContextMock}>
-      {inline ? <InlinePodcastPromo /> : <SecondaryColumnPodcastPromo />}
-    </ServiceContext.Provider>
-  </ToggleContextProvider>
+  <ThemeProvider service="news">
```

Comment:
What's the reason the service is called 'news'?

Reply:
That was an error, I have updated.